conf/machine/include/rpi-base.inc

```diff
@@ -15,6 +15,7 @@ XSERVER = " \
     "
 
 RPI_KERNEL_DEVICETREE_OVERLAYS ?= " \
+    overlays/overlay_map.dtb \
```

Comment:
The log doesn't explain what this actually is. Could you please expand. Also, the commit misses SoB.

Reply:
Added the SoB and a link to the detailed explanation of what overlay_map is for.

Reply:
does this go into overlays/ folder ?

Reply:
Yes. Hence some extra code in the recipe to put this particular file in the overlays folder and not in the root like the rest of the dtb files.
I tested it on the rpi4 board and confirmed that the firmware will only use it when placed in the overlays folder.

Reply:
Thanks for clarifying. I'm good with this.